src/ocean/util/app/DaemonApp.d

```diff
@@ -563,7 +564,23 @@ public abstract class DaemonApp : Application,
 
     private bool statsTimer ( )
     {
-        this.onStatsTimer();
+        try
+        {
+            this.onStatsTimer();
+        }
+        catch (Exception e)
+        {
+            try
+            {
+                Stderr.formatln(
+                        "Exception caught in DaemonApp.statsTimer: {}", e);
```

Comment:
Non-standard indentation!!! :D ;)

Reply:
Please suggest what the standard one is, I had problems fitting this :-)

Reply:
Only joking. It doesn't matter.